test/parallel_api/algorithm/alg.nonmodifying/transform_if.pass.cpp

```diff
@@ -23,7 +23,28 @@
 #include "support/utils.h"
 #include <random>
 
-template <typename Policy, typename Queue>
+using namespace TestUtils;
+
+struct test_transform_if
+{
+    template <typename Policy, typename InputIterator1, typename InputIterator2, typename OutputIterator, typename UnaryOperation, typename Predicate>
+    void
+    operator()(Policy&& exec, InputIterator1 first, InputIterator1 last, InputIterator2 mask, InputIterator2 /* mask_end */, OutputIterator result, OutputIterator /* result_last */, UnaryOperation op, Predicate pred)
+    {
+        const int n = ::std::distance(first, last);
+        auto result_copy = result;
+
+        // Try transform_if
+        //transform_if(exec, first, last, mask, result, op, pred);
+        oneapi::dpl::transform_if(exec, first, last, mask, result, op, pred);
+
+        for (auto iter = first; iter != last; ++iter) {
+            //EXPECT_EQ(*(result + (iter - first)), *(result2 + (iter - first)), "wrong value in output sequence"); 
```

Comment:
Why this code is commented?

Reply:
I think you are looking at a very old single commit rather than the PR as a whole with all this commented code.